src/components/dropdown/dropdown.js

```diff
@@ -77,13 +77,12 @@ class DropDown extends Component {
   /**
    * on click handler
    *
-   * @param {Event} e - keyboard event
    * @returns {void}
    * @memberof DropDown
    */
-  onClick = (e: Event): void => {
-    e.stopPropagation();
```

Comment:
what are the side effects of removing the stopPropagation? we have additional click handlers that we wouldn't want  them to catch the event.
As only drop down is using the menu maybe we can lift the event listener up to dropdown and expose the handler as a callback prop?

Reply:
@JonathanTGold ?

Reply:
i did lift it up and now the click outside event is managed from the dropdwon